components/builder-scheduler/src/server/handlers.rs

```diff
@@ -114,6 +114,42 @@ pub fn group_create(
     Ok(())
 }
 
+pub fn reverse_dependencies_get(
+    req: &mut Envelope,
+    sock: &mut zmq::Socket,
+    state: &mut ServerState,
+) -> Result<()> {
+    let msg: proto::ReverseDependenciesGet = req.parse_msg()?;
+    debug!("reverse_dependencies_get message: {:?}", msg);
+
+    let ident = format!("{}/{}", msg.get_origin(), msg.get_name());
+    let graph = state.graph().read().expect("Graph lock is poisoned");
+    let rdeps = graph.rdeps(&ident);
+    let mut rd_reply = proto::ReverseDependencies::new();
+    rd_reply.set_origin(msg.get_origin().to_string());
+    rd_reply.set_name(msg.get_name().to_string());
+
+    match rdeps {
+        Some(rd) => {
+            let mut rdps = RepeatedField::new();
```

Comment:
Having a variable named `rdps` in the same scope as one named `rdeps` is kinda confusing! Can we name it `short_form_deps` or something similarly more intention-revealing?